src/components/expression/expression.scss

```diff
@@ -44,3 +44,8 @@
     }
   }
 }
+
+.gradient-rect {
+  rx: 10;
```

Comment:
rects don't seem to have rx, ry. Besides attr properties in css won't work I think.